server/scripts/2020TaxSeasonDataScript.php

```diff
@@ -0,0 +1,409 @@
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Center for People in Need data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertBennettMartinLibraryData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+	$reviewerRoleId = 4;
+    
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Bennett Martin data has already been inserted');
+	}
+
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = 7 // Manually obtained from PROD DB
+
+		// Sundays
+		$dates = getWeeklyDatesFromRange('2020-02-02', '2020-03-15')
+		foreach ($dates as $date) {
+			$shiftId = insertShift("$date 13:00:00", "$date 16:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 13:00:00", 8, 8, 100, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 14:00:00", 8, 8, 100, 60, $siteId);
+			$thirdAppointmentTimeId = insertAppointmentTime("$date 15:00:00", 4, 4, 100, 60, $siteId);
+		}
+
+		// Default Site Role Limits
+		insertSiteRoleLimit(1, $siteCoordinatorRoleId, $siteId);
+		insertSiteRoleLimit(1, $greeterRoleId, $siteId);
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting Center for People in Need data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertFStreetCommunityCenterData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+    $reviewerRoleId = 4;
+    
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The F Street Community Center data has already been inserted');
+	}
+	
+	try {
+		$DB_CONN->beginTransaction();
+
+        $siteId = 8; // Manually obtained from PROD DB
+
+		// Tuesdays
+        $dates = getWeeklyDatesFromRange('2020-02-04', '2020-04-07');
+        
+		foreach ($dates as $date) {
+			$shiftId = insertShift("$date 17:00:00", "$date 19:00:00", $siteId);
+
+			$firstAppointmentTimeId = insertAppointmentTime("$date 17:00:00", 8, 8, 100, 60, $siteId);
+			$secondAppointmentTimeId = insertAppointmentTime("$date 18:00:00", 4, 4, 100, 60, $siteId);
+		}
+
+		// Default Site Role Limits
+		insertSiteRoleLimit(1, $siteCoordinatorRoleId, $siteId);
+		insertSiteRoleLimit(1, $greeterRoleId, $siteId);
+
+		$DB_CONN->commit();
+	} catch (Exception $e) {
+		$DB_CONN->rollback();
+		throw new Exception('Failed inserting F Street Community Center data', MY_EXCEPTION);
+		die();
+	}
+}
+
+function insertSoutheastCommunityCollegeData() {
+	GLOBAL $DB_CONN;
+
+	$siteCoordinatorRoleId = 1;
+	$greeterRoleId = 2;
+	$preparerRoleId = 3;
+	$reviewerRoleId = 4;
+
+	$dataAlreadyInserted = true;
+	if ($dataAlreadyInserted) {
+		die('The Southeast Community College Data has already been inserted');
+	}
+ 	
+	try {
+		$DB_CONN->beginTransaction();
+
+		$siteId = insertSite('Southeast Community College', '8800 O Street, #T-102, Lincoln, NE 68520', '402-472-9638', FALSE, FALSE);

```

Comment:
Where did you get the `Lincoln, NE 68520` from? That's not on the flier

Reply:
Deleted. Was preoccupied with the #T-102 extension, and when I saw I should keep that, I just kept everything that was appended. Good catch!